src/index.tsx

```diff
@@ -21,12 +23,20 @@ if (process.env.NODE_ENV !== 'production') {
 const root = ReactDOM.createRoot(document.getElementById('root') as Element);
 const queryClient = new QueryClient();
 
+//just to get token manually until we have login screen
```

Comment:
Can you prepend this comment with "TODO"? this can help quickly finding tech debts like this one later

(and there are IDE extensions supporting TODO/FixMe comments too like [todo-tree]( https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree))